CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+### Changed
+
+### Removed
+
+Remove Meeting Demo app
```

Comment:
Maybe specify that migrating the meeting demo app to aws-sample and give the new link here too.

Reply:
Sure, will update it